external/quarkus_openshift/run_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+################################################################################
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+DEST="$1"
+ARCH=`uname -m`
+
+export MAVEN_OPTS="-Xmx1g"
+
+cd $DEST
+pwd
+
+git clone https://github.com/quarkus-qe/quarkus-openshift-test-suite.git
+cd quarkus-openshift-test-suite
+
+echo "Compile and run quarkus_openshift tests"
+
+if [[ $ARCH =~ "x86" ]]; then
+    oc login --token=bgn5pEjb_1mnyDfAtj6hwOAfRL1WBpLXb6R-C8zsAdQ --server=https://api.digger.cp.fyre.ibm.com:6443 --insecure-skip-tls-verify
```

Comment:
We will have to have a way to parameterize the token and server values (perhaps using environment variables set to the value of whatever the user who wants to run the test requires).

Reply:
Thanks for the comment, I have changed ``` OCP_SERVER``` and```OCP_TOKEN``` into parameters, and tested in Grinder_TKG through adding these two parameters with @llxia 's help. The new test links are: Grinder_TKG 776, 777, and 779. Thanks.